src/app/zones/locales/en-us/index.yaml

```diff
@@ -174,6 +174,7 @@ zones:
   create:
     generalError:
       title: 'Could not create the Zone'
+    invalidNameError: "invalid characters. Valid characters are numbers, lowercase latin letters and '-', '_' symbols."
```

Comment:
Not sure if you meant to capitalize or title case this? (`invalid`)

Reply:
I think this is being further discussed in the comment below so I'm going to resolve this convo.